tabbycat/users/groups.py

```diff
@@ -17,58 +17,79 @@ class BaseGroup:
 
 
 class Equity(BaseGroup):
-    # Permissions to manage conflicts/constraints, view participant info
     name = _("Equity")
     permissions = [
-        Permission.EDIT_ROOMCATEGORIES,
-        Permission.EDIT_ROOMCONSTRAINTS,
+        # View and edit all conflicts
+        Permission.VIEW_ADJ_TEAM_CONFLICTS,
+        Permission.EDIT_ADJ_TEAM_CONFLICTS,
+        Permission.VIEW_ADJ_ADJ_CONFLICTS,
         Permission.EDIT_ADJ_ADJ_CONFLICTS,
+        Permission.VIEW_ADJ_INST_CONFLICTS,
         Permission.EDIT_ADJ_INST_CONFLICTS,
-        Permission.EDIT_ADJ_INST_CONFLICTS,
+        Permission.VIEW_TEAM_INST_CONFLICTS,
         Permission.EDIT_TEAM_INST_CONFLICTS,
+
+        # Room constraints
+        Permission.VIEW_ROOMCONSTRAINTS,
+        Permission.EDIT_ROOMCONSTRAINTS,
+        Permission.VIEW_ROOMCATEGORIES,
+        Permission.EDIT_ROOMCATEGORIES,
+
+        # Participant identity info
         Permission.VIEW_PARTICIPANTS,
-        Permission.VIEW_TEAMS,
-        Permission.VIEW_ADJUDICATORS,
-        Permission.VIEW_ROOMS,
-        Permission.VIEW_INSTITUTIONS,
+        Permission.VIEW_PARTICIPANT_GENDER,
+        Permission.VIEW_PARTICIPANT_CONTACT,
+        Permission.VIEW_PARTICIPANT_DECODED,
+        Permission.VIEW_PARTICIPANT_INST,
         Permission.VIEW_DECODED_TEAMS,
         Permission.VIEW_ANONYMOUS,
-        Permission.VIEW_ADMIN_DRAW,
+
+        # Supporting context
+        Permission.VIEW_INSTITUTIONS,
+        Permission.VIEW_ADJUDICATORS,
     ]
 
 
 class AdjudicationCore(BaseGroup):
     # Permissions to make [preformed] allocations, view feedback, and create motions
     name = _("Adjudication Core")
     permissions = [
-        Permission.EDIT_BASEJUDGESCORES_IND,
-        Permission.EDIT_DEBATEADJUDICATORS,
+        # Feedback
+        Permission.ADD_FEEDBACK,
+        Permission.VIEW_FEEDBACK,
         Permission.EDIT_FEEDBACK_CONFIRM,
```

Comment:
Could we remove this? I'd rather CAs only use the "Ignore" button than "Confirm"